_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -0,0 +1,46 @@
+categories:
+- ROS2
+- MoveIt
+---
+
+With the release of ROS 2 Galactic, we would like to share the new features only available in MoveIt 2. We hope to encourage the community to upgrade to MoveIt 2 as soon as possible. MoveIt 2 is out of beta.
+
+
+|                                                                                                                                                    | MoveIt 1 | MoveIt 2       |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | -------------- |
+| Motion Planning                                                                                                                                    | ✓        | ✓              |
+| Inverse Kinematics                                                                                                                                 | ✓        | ✓              |
+| Perception                                                                                                                                         | ✓        | ✓              |
+| Grasping                                                                                                                                           | ✓        | ✓              |
+| Setup Assistant                                                                                                                                    | ✓        | In Development |
+| Game controller integration for Servo                                                                                                              | ✓        | ✓              |
+| [Probabilistically complete Cartesian Planning](https://github.com/ros-planning/moveit2/tree/main/moveit_demo_nodes/run_ompl_constrained_planning) | ✕        | ✓              |
+| ROS 2 Component Nodes                                                                                                                              | ✕        | ✓              |
+| [Planning for differential drive bases](http://moveit2_tutorials.picknik.ai/doc/mobile_base_arm/mobile_base_arm_tutorial.html)                     | ✕        | ✓              |
+| [Hybrid planning](https://github.com/ros-planning/moveit2/pull/488) (global + local planners)                                                      | ✕        | ✓              |
+| Realtime support via DDS                                                                                                                           | ✕        | ✓              |
+| Improved cross-platform support                                                                                                                    | ✕        | ✓              |
+| Actively developed by PickNik & ROS 2 Technical Steering Committee                                                                                 | ✕        | ✓              |
```

Comment:
The table doesn't look good. If you change `Technical Steering Committee` to `TCS` things fit better
```suggestion
| Actively developed by PickNik & ROS 2 TCS                                                                                                          | ✕        | ✓              |
```

Reply:
I think you meant `TSC`, not `TCS`?

Reply:
I did mean `TSC`